spec/presenters/vendor_api/single_application_presenter_spec.rb

```diff
@@ -28,6 +28,27 @@
     end
   end
 
+  describe 'attributes.hesa_itt_data' do
+    let(:application_form) do
+      create(:completed_application_form, :with_completed_references)
+    end
```

Comment:
Minor: this can be inlined into the `application_choice` let, we don't need it elsewhere.